### PR TITLE
net-libs/miniupnpc: Version bump to 1.9.20151008

### PR DESCRIPTION
**Pull request by luke-jr**
Note: The current versions have severe vulnerabilities.

### CODE REVIEW

**Comment by hasufell** [comment]
@mgorny

File: net-libs/miniupnpc/miniupnpc-1.9.20151008.ebuild [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+EAPI=5
+
+inherit eutils multilib toolchain-funcs
+
+DESCRIPTION="UPnP client library and a simple UPnP client"
+HOMEPAGE="http://miniupnp.free.fr/"
+SRC_URI="http://miniupnp.free.fr/files/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/14"
```
Comment (hasufell) [inline net-libs/miniupnpc/miniupnpc-1.9.20151008.ebuild]: meeep, wrong subslot... must be `15`

```
# objdump -p /usr/lib64/libminiupnpc.so | grep SONAME
  SONAME               libminiupnpc.so.15
```

**Comment by luke-jr** [comment]
Fixed.

**Comment by mgorny** [comment]
Yay, another API change!

**Comment by mgorny** [comment]
Oh wait, the change is actually backwards compatible! Hard to believe, it is. I think we can actually lower the SONAME and subslot to avoid rebuilding everything unnecessarily.

**Comment by hasufell** [comment]
Why does he bump soname when there is no ABI breakage?

**Comment by mgorny** [comment]
He doesn't really do ABI versioning. He just changes API randomly, then changes API version and sets SONAME to match that. Well, it's better than in the days when there was no SONAME.

**Comment by luke-jr** [comment]
So what should I do here?

**Comment by mgorny** [comment]
I'll take it from here, thanks.

**Comment by mgorny** [comment]
The security bug is irrelevant here, there's no related change between the two snapshots. Only some portability #include changes and sameport->localport change.

**Pull request merged**